rpc_jobs/params.yml

```diff
@@ -87,14 +87,6 @@
           default: "{DATA_DISK_DEVICE}"
           description: The data disk to use for VMs (if unset the largest unpartitioned device will be used)
       - string:
-          name: MULTI_NODE_AIO_DIR
-          default: "multi-node-aio"
-          description: |
-            The directory to trigger build.sh from
-            Options:
-              multi-node-aio
-              multi-node-aio-xenial-ansible
-      - string:
           name: "COMPUTE_NODES"
```

Comment:
I'd be inclined towards changing the names of these parameters to be more descriptive/accurate, for example `ADDITIONAL_COMPUTE_NODES`. The parameter descriptions also seem misleading now.

Reply:
Yeah, description was unclear, updating param names.